### ISSUE TITLE
Accessibility Review

### ISSUE THREAD

**Issue by dsamojlenko** [cds-snc/nrcan-energuide-api-poc#53]
Results of a preliminary accessibility review conducted using Axe and Chrome Audits.

## All Pages
- ~~`<html>` element must have a lang attribute~~ https://dequeuniversity.com/rules/axe/3.0/html-has-lang
- ~~`<meta>` Zooming and scaling must not be disabled~~ https://dequeuniversity.com/rules/axe/3.0/meta-viewport
- ~~Content should be contained in a landmark region~~ https://dequeuniversity.com/rules/axe/3.0/region
- ~~Language link should have lang attribute describing the language of the text (ie, `<a lang="fr">Francais</a>`, `<a lang="en">English</a>`)~~
- ~~Tabindex seems to be used incorrectly~~  http://www.punkchip.com/correct-use-of-tabindex/
- Colour contrast ratio issues (there are a bunch - check with Axe)

## Home Page
- ~~#pageWrap ID attribute must be unique~~ https://dequeuniversity.com/rules/axe/3.0/duplicate-id
- ~~`<img>` Images must have alternate text or set as decorative~~ https://dequeuniversity.com/rules/axe/3.0/image-alt

## Search
- ~~#info ID attribute must be unique~~ https://dequeuniversity.com/rules/axe/3.0/duplicate-id
- ~~Heading levels should only increase by one~~
- ~~Help (?) icons - text reads as "Question. A location refers to a region..."  - should set the ? to invisible~~
- ~~Help text should be associated with the form control (possibly with aria-describedby)~~ https://accessibility.18f.gov/forms/

## Search by file ID
- ~~"aria-labelledby" ARIA attributes must conform to valid values (seems like the id "search-by-description" doesn't exist?)~~
	- https://dequeuniversity.com/rules/axe/3.0/aria-valid-attr-value  
	- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

## Results-location
- ~~Heading levels should only increase by one~~ https://dequeuniversity.com/rules/axe/3.0/heading-order
- ~~Table data on this page is incomprehensible - should probably just use proper table markup with accessibility features~~

# Non-accessibility-related comments

## Performance Audit Score: 36/100
Recommendations:
- Enable text compression (server config)
- Minify javascript (vendor.js)

## General Comments
- Page does not have a meta description (SEO)
- Impossible to link directly to french language page equivalents?
- Mailto link at the top goes to Paul
- Should the image on the home page be loaded from Nick's personal github?
